### PR TITLE
add change password URL for univ. of missouri

### PR DESCRIPTION
**Pull request by geoffcline**
### Checklist
- [x] The PR isn't documenting something that would be a common practice among password managers (e.g. minimal length of 6)
- [x] The top-level JSON objects are sorted alphabetically 
- [x] There is no Well-Known URL for Changing Passwords (`https://example.com/.well-known/change-password`)
- [x] There are no [open pull requests](https://github.com/apple/password-manager-resources/pulls) for the same update.

### CODE REVIEW

File: quirks/change-password-URLs.json [hunk 1]
```diff
@@ -24,5 +24,6 @@
     "twitch.tv": "https://www.twitch.tv/settings/security",
     "xfinity.com": "https://customer.xfinity.com/users/me/update-password",
     "yahoo.com": "https://login.yahoo.com/account/change-password",
-    "zoom.us": "https://zoom.us/profile#pwd-form"
+    "zoom.us": "https://zoom.us/profile#pwd-form",
+    "umsystem.edu": "https://password.umsystem.edu/reset/"
```
Comment (rmondello) [inline quirks/change-password-URLs.json:28]: This looks like a reset page. Is there a “change” page, or are changes done via reset?

Comment (geoffcline) [reply quirks/change-password-URLs.json:28]: changes are done via reset, if you visit the page it prompts you for user/pass, but provides options if you don't know the password. 

they require some kind of 2FA to change password (which is not generally required), which is historically why it is called reset

**Comment by rmondello** [comment]
I’m satisfied with that explanation!

**Comment by m33x** [comment]
But what about the alphabetic sorting :-/

**Comment by rmondello** [comment]
I forgot my ABCs! Fixed in da63dde01624173c8fbff0515bab6e69e0a9600c.

**Pull request merged**